server/api/include/irods/rs_get_resource_info_for_operation.hpp

```diff
@@ -8,36 +9,70 @@ struct RsComm;
 
 /// Get preferred-resource information for the specified operation.
 ///
-/// \param[in] _rsComm An rsComm_t connection handle to the server.
+/// \param[in] _comm An RsComm connection handle to the server.
 /// \param[in] _dataObjInp \parblock A DataObjInp describing the data object operation.
 /// - \p objPath: The logical path of the target data object.
 /// - \p condInput: A list of options that influence the results of the operation.
-///     - \p REPL_NUM_KW  - The replica number of the copy to upload.
-///     - \p GET_RESOURCE_INFO_FOR_OPERATION_KW - The operation of interest. The following values are supported:
-///         - "CREATE"
-///         - "WRITE"
-///         - "OPEN"
-///         - "UNLINK"
-///     - \p RESC_NAME_KW - Resource name hint.  Given the name of a root resource, influences voting to select the appropriate replica.
-///     - \p RESC_HIER_STR_KW -  Resource hierarchy hint.  Given a hierarchy string, specifies the desired replica.
-/// \endparblock
-/// \param[out] _out_info \parblock a JSON string with keys "host" and "resource_hierarchy", identifying target server and supplying a
-///                                 resource hierarchy string denoting a specific data object replica for the logical path given as input.
+///     - \p GET_RESOURCE_INFO_FOR_OPERATION_KW: The operation of interest. This option is required. The following
+///       values are supported:
+///         - \p "CREATE"
+///         - \p "WRITE"
+///         - \p "OPEN"
+///         - \p "UNLINK"
+///     - \p RESC_NAME_KW: Resource name hint. Given the name of a root resource, influences voting to select the
```

Comment:
Maybe not worth mentioning, but I believe `DEST_RESC_NAME_KW` would also have a similar effect to `RESC_NAME_KW` for some or all operations.

Reply:
Ok. Here's a question, should we advertise that?

Reply:
After some discussion, we've decided to not mention `DEST_RESC_NAME_KW`.

Users should stick to the documented options and values.